packages/foreman/foreman/foreman.spec

```
@@ -678,6 +678,7 @@ cat > %{buildroot}%{_sysconfdir}/rpm/macros.%{name}-plugin << EOF
 # -n<plugin_name>   Overrides default of gem_name
 %%%{name}_bundlerd_file(n:) \\
 mkdir -p %%{buildroot}%%{%{name}_bundlerd_dir} \\
+echo %%{%{name}_bundlerd_dir}/%%{-n*}%%{!?-n:%%{gem_name}} >> %%%{name}_plugin_files \\
```

Note to self: this and other cases need to be
```suggestion
echo %%{%{name}_bundlerd_dir}/%%{-n*}%%{!?-n:%%{gem_name}} >> %{name}_plugin_files \\
```